contracts/SpokePool.sol

```diff
@@ -55,13 +55,13 @@ abstract contract SpokePool is
     // refunds and slow relays.
     address public hubPool;
 
-    // Address of wrappedNativeToken contract for this network. If an origin token matches this, then the caller can
-    // optionally instruct this contract to wrap native tokens when depositing (ie ETH->WETH or MATIC->WMATIC).
-    WETH9Interface public wrappedNativeToken;
-
-    // Any deposit quote times greater than or less than this value to the current contract time is blocked. Forces
-    // caller to use an approximately "current" realized fee. Defaults to 1 hour.
-    uint32 public depositQuoteTimeBuffer;
+    // Note: storage variables prefixed with DEPRECATED used to be variables that could be set by the cross-domain
```

Comment:
Nit: I would say "the following two storage variables..." since this does not really apply to all deprecated variables, just these two.

Reply:
But doesn't this accurately describe that this applies to "storage variables prefixed with DEPRECATED"?